plugins/home/src/homePageComponents/WelcomeTitle/WelcomeTitle.tsx

```diff
@@ -44,7 +43,7 @@ export const WelcomeTitle = () => {
 
   return (
     <Tooltip title={greeting.language}>
-      <Typography component="span">{`${greeting.greeting}${
+      <Typography variant="h3">{`${greeting.greeting}${
```

Comment:
Hmm? is it not a span anymore? well, I see no reason to keep the span either way.

Reply:
Yeah, it doesn't make sense to me to put it back now that we've settled on using an `H3`.

Reply:
`<Typography component="span" variant="inherit">` works without having a fixed variant.

Reply:
Oh, interesting @micnncim! I'll give that a try, feel like that's a bit more clear as to what the intent is.